docs/guides/fractional-gates.ipynb

```diff
@@ -220,6 +221,15 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Angle constraints\n",
+    "\n",
+    "For the two-qubit $R_{ZZ}(\\theta)$ gate, only angles between $0$ and $\\pi/2$ can be executed on IBM Quantum hardware. If a circuit contains any $R_{ZZ}(\\theta)$ gates with an angle outside of this range, then the standard transpilation pipeline will generally correct this with an appropriate circuit transformation (through the [`FoldRzzAngle`](../api/qiskit-ibm-runtime/transpiler-passes-fold-rzz-angle) pass).  However, any $R_{ZZ}(\\theta)$ gates containing one or more [`Parameter`](../api/qiskit/qiskit.circuit.Parameter)s, the transpiler will assume these parameters will be assigned angles within this range at runtime. However the job will fail if any of the parameter values specified in the PUB submitted to Qiskit Runtime are outside of this range."
```

Comment:
```suggestion
    "For the two-qubit $R_{ZZ}(\\theta)$ gate, only angles between $0$ and $\\pi/2$ can be executed on IBM Quantum hardware. If a circuit contains any $R_{ZZ}(\\theta)$ gates with an angle outside of this range, then the standard transpilation pipeline will generally correct this with an appropriate circuit transformation (through the [`FoldRzzAngle`](../api/qiskit-ibm-runtime/transpiler-passes-fold-rzz-angle) pass).  However, any $R_{ZZ}(\\theta)$ gates containing one or more [`Parameter`](../api/qiskit/qiskit.circuit.Parameter)s, the transpiler will assume these parameters will be assigned angles within this range at runtime. The job will fail if any of the parameter values specified in the PUB submitted to Qiskit Runtime are outside of this range."
```

Reply:
I think this sentence is missing something, maybe a "for" before "any"? (note that I removed some formatting/linking text here to make it more readable, so don't copy/paste it back like this 😄 )

> However, any R_ZZ gates containing one or more Parameters, the transpiler will assume these parameters will be assigned angles within this range at runtime.